metriq_gym/benchmarks/benchmark.py

```diff
@@ -1,17 +1,31 @@
 import argparse
+from collections.abc import Iterable
```

Comment:
As of Python 3.9+, we should be importing this from the `typing` module (not `collections.abc`).

```suggestion
from typing import Iterable
```